configure.ac

```diff
@@ -1,4 +1,4 @@
-AC_INIT([nix],[m4_esyscmd(bash -c "echo -n $(cat ./.version)$VERSION_SUFFIX")])
+AC_INIT([nix],m4_esyscmd(bash -c "echo -n $(cat ./.version)$VERSION_SUFFIX"))
```

Comment:
Is that needed here or is it an accidental change?

Reply:
Good spotting! No, this was because at some point I needed to fix #5832 but you are right that this does not belong here. I'll revert that bit. Thank you.